src/main/histFileMod.F90

```diff
@@ -3076,7 +3078,7 @@ subroutine htape_timeconst(t, mode)
              call ncd_defvar(varname='fates_scmap_levscpf',xtype=ncd_int, dim1name='fates_levscpf', &
                   long_name='FATES size index of the combined pft-size class dimension', units='-', ncid=nfid(t))
              call ncd_defvar(varname='fates_levcacls', xtype=tape(t)%ncprec, dim1name='fates_levcacls', &
-                  long_name='FATES cohort age class lower bound', units='years', ncid=nfid(t))
+                  long_name='FATES cohort age class lower bound (yr)', units='-', ncid=nfid(t))
```

Comment:
Wait why are the units being moved to the long name?

Reply:
Hi @ekluzek the reason for this is that metadata-aware data analysis programs (ferret in particular) get confused when there are two different time dimensions in a given dataset.  The new versions of ferret won't even let us open the history files unless we change the metadata. So this change prevents that from happening and makes it clear to the programs which the relevant time dimension is.

Reply:
Hmmm. OK. That's annoying. It seems like it should realize that there is a different coordinate for time, since the dimension names are different, and the time dimension variables all use the time dimension.

It does look like the CF convention way to handle it would be to add the "coordinate" attribute as follows...

https://cfconventions.org/Data/cf-conventions/cf-conventions-1.7/cf-conventions.html#alternative-coordinates

But, that shouldn't even be necessary since it's a different dimension.

So in the end I think I agree to this change. I might add a note about it though. Thanks for the explanation.